Hakawai/Mentions/HKWMentionsPlugin.m

```diff
@@ -1819,63 +1819,65 @@ - (void)attachEnclosedChooserView:(UIView *)accessoryView origin:(CGPoint)origin
     __weak typeof(self) __self = self;
     __strong __auto_type parentTextView = self.parentTextView;
     CGFloat gapHeight = [parentTextView rectForSingleLineViewportInMode:HKWViewportModeTop].size.height;
+    __weak typeof(parentTextView) __parentTextView = parentTextView;
```

Comment:
Let's name it `weakParentTextView` - I know there's a few places we use __ prefix but it's not good.

Reply:
Good point. Actually, since `self.parentTextView` is already weak reference, I capture it in the block directly instead.